src/devices/Vl53L1X/Vl53L1X.cs

```diff
@@ -0,0 +1,1022 @@
+                BinaryPrimitives.WriteUInt16BigEndian(writeArray, (byte)Registers.VL53L1X_I2C_SLAVE__DEVICE_ADDRESS);
+                writeArray[2] = newAddress;
+                defaultAddressDevice.Write(writeArray);
+                Thread.Sleep(10);
+            }
+            catch (IOException ex)
+            {
+                throw new IOException($"Can't change I2C Address to {newAddress}", ex);
+            }
+            finally
+            {
+                defaultAddressDevice.Dispose();
+            }
+
+            _i2CDevice = I2cDevice.Create(new I2cConnectionSettings(busId, newAddress));
+        }
+
+        private void InitSensor()
+        {
+            for (byte register = 0x2D; register <= 0x87; register++)
+            {
+                WriteRegister(register, _vl53L1XDefaultConfiguration[register - 0x2D]);
+            }
+
+            StartRanging();
+            WaitForDataReady();
+
+            ClearInterrupt();
+            StopRanging();
+            WriteRegister((ushort)Registers.VL53L1X_VHV_CONFIG__TIMEOUT_MACROP_LOOP_BOUND, 0x9); /* two bounds VHV */
+            WriteRegister(0xB, 0); /* start VHV from the previous temperature */
+        }
+
+        /// <summary>
+        ///     This function starts the ranging distance operation which is continuous.
+        ///     The clear interrupt has to be done after each "get data" to allow the interrupt to be raised when the next data are
+        ///     ready.
+        ///     1 = active high (default), 0 =active low.
+        ///     If required, use <see cref="SetInputPolarity" /> to change the interrupt polarity.
+        /// </summary>
+        public void StartRanging()
+        {
+            ClearInterrupt();
+            WriteRegister((ushort)Registers.SYSTEM__MODE_START, 0x40);
+            _rangingInitialized = true;
+        }
+
+        /// <summary>
+        ///     This function stops the ranging.
+        /// </summary>
+        public void StopRanging()
+        {
+            WriteRegister((ushort)Registers.SYSTEM__MODE_START, 0x00);
+            _rangingInitialized = false;
+        }
+
+        /// <summary>
+        ///     This function returns whether to raning is active or not
+        /// </summary>
+        /// <returns></returns>
+        public bool IsRangingInitialized()
+        {
+            byte systemMode = ReadByte((ushort)Registers.SYSTEM__MODE_START);
+            return systemMode == 0x40;
+        }
+
+        /// <summary>
+        ///     This function checks if the new ranging data are available by polling the dedicated register.
+        /// </summary>
+        /// <returns>True if data is ready, otherwise false</returns>
+        private bool IsDataReady()
+        {
+            byte temp = ReadByte((byte)Registers.GPIO__TIO_HV_STATUS);
+            return temp != 0; // (temp & 1) == inputPolarity;
+        }
+
+        /// <summary>
+        ///     This function programs the interrupt polarity
```

Comment:
note: it is best to finish comments with a period (dot)
And also to have them with only 1 space to start.
If you can adjust everywhere, it will be great

Reply:
All comments have been adjusted